### DISCUSSION TITLE
Problem with Lc0 compiled and used under Mac OS

### DISCUSSION THREAD

**Discussion by GiordanoVicoli** [LeelaChessZero/lc0#2008] [Q&A]
Hi,

I compiled Lc0 in my macbook M1 Pro I inserted the executable into one of my Mac OS applications where I have other engines compiled (different versions of Stockfish and komodo). I access these motors via UCI commands and everything seems to work. I tried to do the same thing with Lc0 but I get some errors one of which I don't understand and is the following: PROCESS ENGINE OUTPUT: *****> error The Metal backend requires a network file. How can I solve the problem?

Thanks,
Giordano

**Comment by mooskagh** [comment]
In order to run Lc0, you need a network file. Pre-compiled versions (e.g. for Windows) usually include one, but if you compile one yourself, you have to download it.

For example, you can download [t1-512x15x8h-distilled-swa-3395000 network](https://storage.lczero.org/files/networks-contrib/t1-512x15x8h-distilled-swa-3395000.pb.gz), and put it to the same directory as your compiled `lc0`. That should work.
